packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateModule.h

```diff
@@ -1,7 +1,7 @@
 //  Copyright © 2018 650 Industries. All rights reserved.
 
-#import <EXCore/EXExportedModule.h>
-#import <EXCore/EXModuleRegistryConsumer.h>
+#import <UMCore/UMExportedModule.h>
+#import <UMCore/UMModuleRegistryConsumer.h>
 
-@interface EXModuleTemplateModule : EXExportedModule <EXModuleRegistryConsumer>
+@interface EXModuleTemplateModule : UMExportedModule <UMModuleRegistryConsumer>
```

Comment:
No module, thats Android

```suggestion
@interface EXModuleTemplate : UMExportedModule <UMModuleRegistryConsumer>
```